tests/zfs-tests/tests/functional/rsend/rsend_012_pos.ksh

```diff
@@ -138,9 +138,7 @@ done
 
 for vol in "$POOL/vol" "$POOL/$FS/vol" ; do
 	rand_set_prop $vol checksum "on" "off" "fletcher2" "fletcher4" "sha256"
-	rand_set_prop $vol compression "on" "off" "lzjb" "gzip" \
-		"gzip-1" "gzip-2" "gzip-3" "gzip-4" "gzip-5" "gzip-6"   \
-		"gzip-7" "gzip-8" "gzip-9"
+	rand_set_prop $vol compression "off" "lzjb" "gzip" "lz4"
```

Comment:
add zle? (separate PR?)

Reply:
ZLE with random writen data (which is often used) is basically the same as `compression=off`